config/locales/wagon.de.yml

```
@@ -1323,15 +1323,15 @@ de:
         redirect_existing_email: Es existiert bereits ein Login für diese E-Mail. Melde dich hier an.
         existing_membership_notice: >-
           Du besitzt bereits eine SAC-Mitgliedschaft. Wenn du diese anpassen möchtest,
-          kontaktiere bitte die SAC Geschäftsstelle.
-        existing_family_notice: Du ist einer Familie zugeordnet. Kontaktiere bitte die SAC Geschäftsstelle.
+          kontaktiere bitte die SAC-Geschäftsstelle.
+        existing_family_notice: Du ist einer Familie zugeordnet. Kontaktiere bitte die SAC-Geschäftsstelle.
         signed_up_notice: Deine Anmeldung wurde erfolgreich gespeichert.
 
       agreement_link:
         agb_text: Ich habe die %{link} gelesen und stimme diesen zu
         statutes_text: Ich habe die %{link} gelesen und stimme diesen zu
         sektion_statuten_text: Ich habe die %{link} gelesen und stimme diesen zu
-        contribution_regulations_text: Ich habe das %{link} gelesen und stimme diesen zu
+        contribution_regulations_text: Ich habe das %{link} gelesen und stimme diesem zu
         data_protection_text: Ich habe die %{link} gelesen und stimme diesen zu
```

Das scheint auch ein Typo zu sein.

```suggestion
        data_protection_text: Ich habe die %{link} gelesen und stimme dieser zu
```